force-app/components/courseInvitation/lwc/courseInvitation/courseInvitation.js

```diff
@@ -81,7 +113,7 @@ export default class CourseInvitation extends NavigationMixin(LightningElement)
     }
 
     get isConfirmDisabled() {
-        return this.recipients.length === 0;
+        return (this.recipients.length === 0 || this.recipients.length > 2);
```

Comment:
```suggestion
    return this.recipients.length === 0; 
```

endre tilbake fordi import-knapp blir skjult